chevah/keycert/ssh.py

```diff
@@ -1183,19 +1183,19 @@ def _fromString_PRIVATE_SSHCOM(cls, data, passphrase):
         cipher_type, rest = common.getNS(rest)
         encrypted_blob, _ = common.getNS(rest)
 
-        if cipher_type.lower() not in ['none', '3des-cbc']:
+        if cipher_type.lower() not in [b'none', b'3des-cbc']:
             raise BadKeyError(
-                'Encryption method not supported: %s' % (
-                    cipher_type))
+                'Encryption method not supported: %r' % (
+                    cipher_type[:30]))
 
         encryption_key = None
-        if cipher_type.lower() == '3des-cbc':
+        if cipher_type.lower() == b'3des-cbc':
```

Comment:
is a bit strange that we call 'lower' on a set of bytes... but I can see that py3  still has the lower method for bytes.